### PR TITLE
Sync docs with current status of `upgrade` command.

### PR DESCRIPTION
**Pull request by szymonrybczak**
Recently the `upgrade` command in React Native CLI [was refactored](https://github.com/react-native-community/cli/pull/1879) to check the current and the latest version and give to user link to the Upgrade Helper with the version provided in URL. This PR sync docs with the current job that actually `upgrade` command does.

### CODE REVIEW

File: docs/upgrading.md [hunk 1]
```diff
@@ -13,57 +13,6 @@ Upgrading your Expo project to a new version of React Native requires updating t
 
 Because typical React Native projects are essentially made up of an Android project, an iOS project, and a JavaScript project, upgrading can be rather tricky. There's currently two ways for upgrading your React Native project: by using [React Native CLI](https://github.com/react-native-community/cli) or manually with [Upgrade Helper](https://react-native-community.github.io/upgrade-helper/).
```
Comment (Pranav-yadav) [inline docs/upgrading.md:14]: Here we've mentioned there are two ways, so you should update this and the wording around it accordingly, at the end the flow of this doc should be according to the headings and suheadings👍

Comment (szymonrybczak) [reply docs/upgrading.md:14]: Good catch!

File: docs/upgrading.md [hunk 2]
```diff
@@ -74,6 +23,8 @@ You first need to select from and to which version you wish to upgrade, by defau
 
 💡 Major updates will show an "useful content" section on the top with links to help you out when upgrading.
 
+> Or you can run the `npx react-native upgrade`, which will automatically check your current version and the latest version available and will show you the link to the Upgrade Helper page with the versions already selected.
+
 #### 2. Upgrade dependencies
 
 The first file that is shown is the `package.json`, it's good to update the dependencies that are showing in there. For example, if `react-native` and `react` appears as changes then you can install it in your project by running `yarn add`:
```
Comment (Pranav-yadav) [inline docs/upgrading.md:30]: Again not related to this PR.
Could you please also add the `npm` equivalent (`npm install`) for `yarn add` throughout this doc?
e.g.: L34, L35

You should utilize the TabItems and Tab components, just search for TabItem in vscode/in your editor :)

**Review by Pranav-yadav — COMMENTED** [review]
Some changes suggested are not related to this PR :)

File: docs/upgrading.md [hunk 3]
```diff
@@ -74,6 +23,8 @@ You first need to select from and to which version you wish to upgrade, by defau
 
 💡 Major updates will show an "useful content" section on the top with links to help you out when upgrading.
 
+> Or you can run the `npx react-native upgrade`, which will automatically check your current version and the latest version available and will show you the link to the Upgrade Helper page with the versions already selected.
```
[the remark this answers is not in the document]
Comment (szymonrybczak) [reply docs/upgrading.md:26]: I think tip will be great 👍 Updated ✅

File: docs/upgrading.md (hunk 2 above)
Comment (szymonrybczak) [reply docs/upgrading.md:30]: Sure, let me create PR :)

File: docs/upgrading.md [hunk 4]
```diff
@@ -74,6 +23,8 @@ You first need to select from and to which version you wish to upgrade, by defau
 
 💡 Major updates will show an "useful content" section on the top with links to help you out when upgrading.
 
+> **_Tip:_** Or you can run the `npx react-native upgrade`, which will automatically check your current version and the latest version available and will show you the link to the Upgrade Helper page with the versions already selected.
```
[the remark this answers is not in the document]
Comment (szymonrybczak) [reply docs/upgrading.md]: Oh yeah, fixed.

**Comment by szymonrybczak** [comment]
Can we get this merged? 🙏

**Pull request merged**